<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConvertMapFileDialog</class>
 <widget class="QDialog" name="ConvertMapFileDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>445</width>
    <height>164</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>422</width>
    <height>145</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Convert map file</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0" colspan="3">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="lSourceFile">
       <property name="text">
        <string>Source file</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="pbSourceFile">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="lDestinationFile">
       <property name="text">
        <string>Destination file</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="leDestinationFile"/>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="pbDestinationFile">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="leSourceFile"/>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <spacer name="hsConvertLeft">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>166</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="pbConvert">
     <property name="text">
      <string>Convert</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <spacer name="hsConvertRight">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>165</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="1" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QCheckBox" name="cbSortByType">
     <property name="text">
      <string>Sort the layers by item type</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ConvertMapFileDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>440</x>
     <y>159</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>144</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ConvertMapFileDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>417</x>
     <y>136</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>144</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbConvert</sender>
   <signal>clicked()</signal>
   <receiver>ConvertMapFileDialog</receiver>
   <slot>onConvert()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>256</x>
     <y>130</y>
    </hint>
    <hint type="destinationlabel">
     <x>104</x>
     <y>99</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbSourceFile</sender>
   <signal>clicked()</signal>
   <receiver>ConvertMapFileDialog</receiver>
   <slot>onSfBrowse()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>439</x>
     <y>34</y>
    </hint>
    <hint type="destinationlabel">
     <x>289</x>
     <y>79</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbDestinationFile</sender>
   <signal>clicked()</signal>
   <receiver>ConvertMapFileDialog</receiver>
   <slot>onDfBrowse()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>406</x>
     <y>64</y>
    </hint>
    <hint type="destinationlabel">
     <x>316</x>
     <y>94</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>leDestinationFile</sender>
   <signal>textChanged(QString)</signal>
   <receiver>ConvertMapFileDialog</receiver>
   <slot>onLeChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>172</x>
     <y>60</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>116</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>leSourceFile</sender>
   <signal>textChanged(QString)</signal>
   <receiver>ConvertMapFileDialog</receiver>
   <slot>onLeChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>141</x>
     <y>31</y>
    </hint>
    <hint type="destinationlabel">
     <x>42</x>
     <y>120</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onConvert()</slot>
  <slot>onSfBrowse()</slot>
  <slot>onDfBrowse()</slot>
  <slot>onLeChanged()</slot>
 </slots>
</ui>
